### DISCUSSION TITLE
how to trigger the input using alpine ?

### DISCUSSION THREAD

**Discussion by ibrahem-kamal** [tasteui/tasteui#498] [Q&A]
so for example i have a custom design for date input that to apply with tallstack i need to override the entire blade component so i wanted to add the tallui date component as hidden input and when the user clicks on my input i trigger the click on the hidden component but looks like x-ref on the tall component don't take effect. any ideas or suggestion ?

**Comment by devajmeireles** [comment]
I couldn't understand.

**Comment by ibrahem-kamal** [comment]
basically i want to have a hidden tall component and when the user click on another input i trigger the click function on this hidden component

**Reply by devajmeireles** [reply]
Sorry about being late. This is something not related to TSUI. I suggest you look for support in the communities about Livewire.
